docs/user/openstack/deploy_on_ovs_dpdk.md

```diff
@@ -0,0 +1,94 @@
+# Deploying an OpenShift Cluster onto OpenStack with OVS-DPDK
+
+
+An OpenStack deployment with ovs-dpdk enabled provides network performance improvements.
+
+Deploying an OpenShift cluster onto an Openstack deployment with ovs-dpdk, is a straight forward process. 
+
+## Selecting hugetable size and cpu policy for cluster vms.
+When deploying vms on to this type of PaaS, the correct hugetable size and cpu pinnings need to be configured for the vms. 
+
+
+This maybe accomplished in two ways: 
+
+
+2) The flavors used for the OCP are defined with appropriate values. For example: 
+   ```
+   $opensack openstack flavor create dpdk_flavor --ram 16384 --disk 45 --vcpus 4
+   $ openstack flavor set \
+    --property dpdk=true \
+    --property hw:cpu_policy=dedicated \
+    --property hw:mem_page_size=large \
+    dpdk_flavor.small
+   ```
+
+3) The image properties maybe be defined in the install-config.yaml using [platform.openstack.
+   clusterOSImageProperties][customization.md]. For example:
+   ```
+   platform:
+      openstack:
+         clusterOSImageProperties:
+            - hw_cpu_policy: dedicated
+            - hw_mem_page_size: 1GB
+   ```
+   The flavor used to deploy the images must allow the image property values selected. For example, if you attempt 
+   to deploy a vm using an iamge with the property hw_mem_page_size: 1GB, but the flavor being used is set with 
+   --property hw:meme_page_size=small, the deployment of the vm will fail because the flavor does not allow for 
+   using a 1GB page size. The flavor property would need to either not be set, which is the same as 
+   hw:mem_page_size=any, or the flavor property should be set to hw:mem_page_size=large.
+
+
+## OpenShift cluster deployment:
+### Create aggregate and flavor
+Create a ‘dpdk’ aggregate group and flavor to use for the OpenShift cluster, [per the OSP documentation][1]
+First, [create the aggregate group:
+```
+$ openstack aggregate create dpdk_group
+$ openstack aggregate add host dpdk_group [compute-host]
+$ openstack aggregate set --property dpdk=true dpdk_group
+```
```

Comment:
Do we need to do this if we're using the `platform.openstack.clusterOSImageProperties` approach described above?